build.gradle

```diff
@@ -241,6 +268,116 @@ rootProject.tasks.named("processSmokeTestResources") {
   duplicatesStrategy = 'include'
 }
 
+tasks.withType(Jar).all {
+  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+}
+
+distTar {
+  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+}
+
+distZip {
+  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+}
+
 wrapper {
     distributionType = Wrapper.DistributionType.ALL
 }
+
+subprojects {
+  apply plugin: 'java'
+  apply plugin: 'eclipse'
+  apply plugin: 'idea'
+  apply plugin: "org.sonarqube"
+  apply plugin: 'info.solidsoft.pitest'
+  apply plugin: 'org.springframework.boot'
+  apply plugin: 'io.spring.dependency-management'
+  apply plugin: 'com.github.ben-manes.versions'
+
+  configurations {
+    testUnitImplementation.extendsFrom testImplementation
+    testUnitImplementation.extendsFrom runtimeOnly
+
+    testIntegrationImplementation.extendsFrom testImplementation
+    testIntegrationImplementation.extendsFrom runtimeOnly
+  }
+
+  dependencies {
+
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
```

Comment:
s this dependency needed? The commissioning service doesn't use a database so this may be able to be removed.

Reply:
removed